packages/0002-Make-Cache-Control-headers-configurable.patch

```diff
@@ -0,0 +1,81 @@
+From c7b8d51d71af638f9a4a8eedd3d597ad56882a64 Mon Sep 17 00:00:00 2001
+From: Nicolas Trangez <ikke@nicolast.be>
+Date: Wed, 13 Mar 2019 15:00:14 +0100
+Subject: [PATCH 2/2] Make `Cache-Control` headers configurable
+
+The default SaltAPI CORS settings don't allow the client to set a
+`Cache-Control` header, which this application does unconditionally.
+
+Either SaltAPI should be changed (because setting the `no-cache` header
+value makes a lot of sense), or we should take a risk and disable it at
+the application side.
+
+This commit makes this configurable.
+---
+ saltgui/config.js                |  1 +
+ saltgui/static/scripts/api.js    | 12 ++++++++++--
+ saltgui/static/scripts/router.js |  7 ++++++-
+ 3 files changed, 17 insertions(+), 3 deletions(-)
+
+diff --git a/saltgui/config.js b/saltgui/config.js
+index d625e63..172a09b 100644
+--- a/saltgui/config.js
++++ b/saltgui/config.js
+@@ -1,3 +1,4 @@
+ window.SALTGUI_CONFIG = {
+     // API_URL: "",
++    // enableCacheControl: true,
+ };
+diff --git a/saltgui/static/scripts/api.js b/saltgui/static/scripts/api.js
+index c25d219..588f05f 100644
+--- a/saltgui/static/scripts/api.js
++++ b/saltgui/static/scripts/api.js
+@@ -8,8 +8,12 @@ class HTTPError extends Error {
+ 
+ 
+ class API {
+-  constructor(apiurl="") {
++  constructor(apiurl="", options={}) {
+     this.APIURL = apiurl;
++    this.enableCacheControl =
++        typeof options.enableCacheControl === "undefined"
```

Comment:
I'm not sure the `typeof` is needed here. Also, this ternary could be rewritten:

```
this.enableCacheControl = options.enableCacheControl === undefined || options.enableCacheControl
```